src/application/router/routes.ts

```diff
@@ -127,17 +135,18 @@ const routes: RouteRecordRaw[] = [
     meta: {
       pageTitleI18n: 'pages.joinTeam',
       discardTabOnLeave: true,
+      authRequired: true,
     },
   },
   {
-    name: 'join',
-    path: '/join/:hash',
-    component: JoinPage,
+    name: 'authorization',
+    path: '/auth',
+    component: AuthorizationPage,
     props: route => ({
-      invitationHash: String(route.params.hash),
+      redirect: String(route.query.redirect),
     }),
     meta: {
-      pageTitleI18n: 'pages.joinTeam',
+      pageTitleI18n: 'pages.authorization',
       discardTabOnLeave: true,
     },
   },
```

Comment:
Why remove join?

Reply:
this was duplicated join route, i replaced duplicated join with authorization 
![image](https://github.com/codex-team/notes.web/assets/130844513/6621c339-fc9f-4454-bc4f-79ae00ed08b5)